### PR TITLE
ci: test on node 18 and 20

### PR DESCRIPTION
**Pull request by chingor13**

### CODE REVIEW

**Comment by regseb** [comment]
There's also the Node 16 in the [`.github/workflows/release-please.yaml`](https://github.com/google-github-actions/release-please-action/blob/9178887cf62c823df78c936f394ebb8053602ef6/.github/workflows/release-please.yaml#L15) file. I think you should upgrade to Node 20.

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -30,6 +30,6 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-node@v1
         with:
-          node-version: 16
+          node-version: 18
```
Comment (regseb) [inline .github/workflows/ci.yaml:33]: In the current version, there is `[14, 16]` and `16`. I would have used versions `[18, 20]` and **`20`**.

```suggestion
          node-version: 20
```

**Comment by regseb** [comment]
There is `'node16'` in [`action.yml`](https://github.com/google-github-actions/release-please-action/blob/9963ea2/action.yml#L171) file. I don't know how this file works, but perhaps this version should also be changed.

**Comment by chingor13** [comment]
> There is `'node16'` in [`action.yml`](https://github.com/google-github-actions/release-please-action/blob/9963ea2/action.yml#L171) file. I don't know how this file works, but perhaps this version should also be changed.

That will actually change the runtime for the action, which we will do separately before release.

**Review by bcoe — APPROVED** [review]

**Review by SurferJeffAtGoogle — APPROVED** [review]

**Pull request merged**